### PR TITLE
Add `email` and `commits` columns to the `system.contributors` table

### LINKED ISSUE #52192: Add email and commits columns to system.contributors table
**Use case**

```sql
SELECT * FROM system.contributors
```

```
┌─name─────────────┬─email───────────────────────────────────┬─commits─┐
└──────────────────┴─────────────────────────────────────────┴─────────┘
```

**Describe the solution you'd like**

Right now we have `system.contributors` table with just one `name` column which maybe used for variety of internal tasks. It would we great if we could also have `email` and `commits` columns. 

**Describe alternatives you've considered**

Using git commands directly to get the list of contributors.

### PR DESCRIPTION
**Pull request by sirvickr**
Closes [52192](https://github.com/ClickHouse/ClickHouse/issues/52192)

### Changelog category (leave one):
- New Feature

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Added new columns to the `system.contributors` table: `email` to distinguish contributors with the same name and to filter them by their emails and `commits` to display the contributor's commits number.

### Documentation entry for user-facing changes
- [x] Documentation is written (mandatory for new features)

### CODE REVIEW

**Comment by alexey-milovidov** [comment]
I'm not sure about the number of commits.
Different people have different styles of committing.
We don't do squashing (to keep the history and allow easy searching in the commits graph).

**Comment by alexey-milovidov** [comment]
While email is easily accessible from the git log, having it in the table exposed directly may impose more questions than it should.

**Comment by alexey-milovidov** [comment]
TLDR, looks like we don't need this change :cry:

**Comment by UnamedRus** [comment]
> email to distinguish contributors with the same name

Calculate hash of email instead?

**Comment by alexey-milovidov** [comment]
> Calculate hash of email instead?

Too smart, we can do nothing instead.

**Comment by rvasin** [comment]
@alexey-milovidov we had an internal meeting. The PR is not critical for us. You may close this PR. We can use git log for our tasks. The PR was rather an warm up/educational task. Such as Viktor could learn system tables programming before starting to work more useful tasks with system tables like #43995.
